src/test/java/StringTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class StringTest {
+    @Test
+    void testSplit() {
+        final String targetString = "1,2";
+        final String[] split = targetString.split(",");
```

Comment:
```suggestion
        String[] result = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?